mm/2s2h/Enhancements/DifficultyOptions/JinxedTimer.cpp

```diff
@@ -0,0 +1,24 @@
+#include <libultraship/bridge.h>
+#include "2s2h/GameInteractor/GameInteractor.h"
+#include "2s2h/ShipInit.hpp"
+
+#define CVAR_NAME "gEnhancements.DifficultyOptions.JinxedTimer"
+#define CVAR CVarGetInteger(CVAR_NAME, 60)
+
+void RegisterJinxedTimer() {
+    COND_ID_HOOK(ShouldActorUpdate, ACTOR_PLAYER, CVAR < 60, [](Actor* actor, bool* should) {
+        if (gSaveContext.jinxTimer == 0) {
+            return;
+        }
+
+        if (CVAR == 0) {
+            gSaveContext.jinxTimer = 0;
+        } else {
+            if (gSaveContext.jinxTimer > CVAR * 20) {
+                gSaveContext.jinxTimer = CVAR * 20;
+            }
+        }
```

Comment:
Unless I'm mistaken, your final condition should work fine with 0, so you can just replace this whole block with
```suggestion
        if (gSaveContext.jinxTimer > CVAR * 20) {
            gSaveContext.jinxTimer = CVAR * 20;
        }
```